docs/releasing.md

```diff
@@ -67,42 +19,45 @@ Cutting a Release
     export NEXT_VERSION=X.Y.Z-SNAPSHOT
     ```
 
-3. Set environment variables with your [Sonatype credentials][sonatype_issues].
-
-    ```
-    export SONATYPE_NEXUS_USERNAME=johnbarber
-    export SONATYPE_NEXUS_PASSWORD=`pbpaste`
-    ```
-
-4. Update, build, and upload:
+3. Update versions:
 
     ```
     sed -i "" \
-      "s/version = '.*'/version = '$RELEASE_VERSION'/g" \
-      build.gradle
+      "s/version = \".*\"/version = \"$RELEASE_VERSION\"/g" \
+      build.gradle.kts
     sed -i "" \
       "s/\"com.squareup.okhttp3:\([^\:]*\):[^\"]*\"/\"com.squareup.okhttp3:\1:$RELEASE_VERSION\"/g" \
       `find . -name "README.md"`
     sed -i "" \
       "s/\/com.squareup.okhttp3\/\([^\:]*\)\/[^\/]*\//\/com.squareup.okhttp3\/\1\/$RELEASE_VERSION\//g" \
       `find . -name "README.md"`
-    ./gradlew clean publishAllPublicationsToMavencentralRepository
     ```
 
-5. Visit [Sonatype Nexus][sonatype_nexus] to promote (close then release) the artifact. Or drop it
-   if there is a problem!
-
-6. Tag the release, prepare for the next one, and push to GitHub.
+4. Tag the release and push to GitHub.
 
     ```
     git commit -am "Prepare for release $RELEASE_VERSION."
     git tag -a parent-$RELEASE_VERSION -m "Version $RELEASE_VERSION"
+    git push && git push --tags
+    ```
+
+5. Wait for [GitHub Actions][github_actions] to start building the release.
```

Comment:
If you release from tags rather than commits you can eliminate this step. See Turbine, for example.

Reply:
Ooooh I want that. Next PR!